conf/application.conf

```diff
@@ -76,6 +76,13 @@ microservice {
         port     = 9514
         basePath = ""
       }
+
+      email {
+        protocol = "http"
```

Comment:
Do we have the PRs for app-conf-{env} we will need to over ride these values

Reply:
Added in app-config-base for returns